workspace_hack/Cargo.toml

```diff
@@ -64,7 +64,7 @@ serde_json = { version = "1", features = ["raw_value"] }
 smallvec = { version = "1", default-features = false, features = ["write"] }
 subtle = { version = "2" }
 time = { version = "0.3", features = ["local-offset", "macros", "serde-well-known"] }
-tokio = { version = "1", features = ["fs", "io-std", "io-util", "macros", "net", "process", "rt-multi-thread", "signal", "test-util"] }
+tokio = { version = "1", features = ["full", "test-util"] }
```

Comment:
Is `= "full"` really needed? I see that `tokio-epoll-uring` [enables the `"full"` feature](https://github.com/neondatabase/tokio-epoll-uring/blob/952b01098ad2a870a1a90dbb48c8794792d73cbd/tokio-epoll-uring/Cargo.toml#L14), so this is more of an upstream concern.

Reply:
https://github.com/neondatabase/tokio-epoll-uring/pull/31

Reply:
https://github.com/neondatabase/neon/pull/6118/commits/26933e2746384c4524d3a7fbf0c7a528a350708e